Documentation/gettingstarted/policy-creation.rst

```diff
@@ -47,32 +48,42 @@ To observe policy audit messages, follow these steps:
          If you installed Cilium via ``helm install``, then you can use ``helm
          upgrade`` to enable Policy Audit Mode:
 
-         .. parsed-literal::
+         .. code-block:: shell-session
```

Comment:
Same remark as Joe's for the `.. pased-literal`, here and for every block with RST substitutions (`|SOMETHING|`).

Reply:
reverted this change back to "code-block..."